lib/buildmodel.js

```diff
@@ -135,6 +135,81 @@ const BuildModel = class {
     stream(config, response) {
         return this.executor.stream(config, response);
     }
+
+    /**
+     * Get a build based on id
+     * @param  {String}   id                The key of the record to retrieve
+     * @return {Function} callback
+     */
+    get(id, callback) {
+        const config = {
+            table: 'builds',
+            params: {
+                id
+            }
+        };
+
+        this.datastore.get(config, (err, data) => {
+            if (err) {
+                return callback(err);
+            }
+            if (!data) {
+                return callback(new Error('Record does not exist'));
+            }
+
+            return callback(data);
+        });
+    }
+
+    /**
+     * List builds with pagination
+     * @param  {Object}   paginate                  Config object
+     * @param  {Number}   paginate.count     Number of items per page
+     * @param  {Number}   paginate.page      Specific page of the set to return
+     * @return {Function} callback
+     */
+    list(paginate, callback) {
+        const config = {
+            table: 'builds',
+            params: {},
+            paginate: {
+                count: paginate.count,
+                page: paginate.page
+            }
+        };
+
+        this.datastore.scan(config, (err, data) => {
+            if (err) {
+                return callback(err);
+            }
+
+            return callback(null, data);
+        });
+    }
+
+    /**
+     * Update a build
+     * @param  {String}    id             The key of the record to retrieve
+     * @param  {Object}    data           The new data object to update with
+     * @return {Function} callback
+     */
+    update(id, data, callback) {
```

Comment:
I think it's better to still pass `id` and `data` in a `config` object so that the arrity of all the functions are the same